src/main/fc/fc_init.c

```diff
@@ -235,7 +235,7 @@ void spiPreInit(void)
 #ifdef USE_MAG_SPI_AK8963
     spiPreInitCs(IO_TAG(AK8963_CS_PIN));
 #endif
-#ifdef RTC6705_CS_PIN // XXX VTX_RTC6705? Should use USE_ format.
+#if defined(USE_VTX_RTC6705) && !defined(USE_VTX_RTC6705_SOFTSPI)
```

Comment:
Because `USE_VTX_RTC6705_SOFTSPI` does not use `RTC6705_CS_PIN`. If we want to preinit CS for `USE_VTX_RTC6705_SOFTSPI`, the pin definition is `RTC6705_SPILE_PIN`. (We could either rename `RTC6705_SPILE_PIN` to `RTC6705_CS_PIN`, or add an extra conditional preinit for `USE_VTX_RTC6705_SOFTSPI`. Thoughts?

Reply:
Hmm. I'd rather fix the pin def for softspi to be consistent with others.

Reply:
Agreed. Amended.